share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -1,6 +1,7 @@
 stages: [ "generate", "build", "publish" ]
 
 variables:
+  SPACK_CI_ENABLE_STACKS: /^(windows-vis)$/
```

Comment:
This bit has to go before merging, and probably before any final testing in staging as well.  Since this PR affects so much non-windows functionality, it will be good to make sure everything else is still building.